website/components/TypeDefinition.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { css } from 'emotion';
 import Code from '@leafygreen-ui/code';
 import { InlineCode } from '@leafygreen-ui/typography';
+import { spacing } from '@leafygreen-ui/tokens';
 import { readmeDepthMap, HeadingType } from './PropTable';
 
 function TypeDefinition({ markdownAst, readme }) {
```

Comment:
this is lacking types. should probably figure out what's wrong with the TS configuration

Reply:
fixed, but still confused re TS configuration. I thought I already solved for this 🤔